pages/en/lb4/Server.md

```diff
@@ -0,0 +1,220 @@
+---
+lang: en
+title: 'Server'
+keywords: LoopBack 4.0, LoopBack 4
+tags:
+sidebar: lb4_sidebar
+permalink: /doc/en/lb4/Server.html
+summary:
+---
+
+Server is a system that enters a working state of doing something by starting it, 
+and exits the working state by stopping it.
+Usually a Server is responsible for interacting with events outside, process incoming things
+and give corresponding responses.
+
```

Comment:
I'm going to grammar nazi all over this, so please don't mind me.
This should be:
`Usually a Server is responsible for interacting with events outside, processing incoming things
and giving corresponding responses.`